cleanlab/regression/rank.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np 
+
+def get_label_quality_score(
```

Comment:
```suggestion
def get_label_quality_scores(
```

Reply:
You want to stick with the conventions used elsewhere in the package, not invent your own except where it's strictly necessary.

Reply:
modified in commit: [0503595](https://github.com/krmayankb/cleanlab/pull/1/commits/05035950d782fee4adde8da6c44afb47953fb5db)